src/styles/utils/normalize.scss

```diff
@@ -0,0 +1,5 @@
+body,
+div {
+  box-sizing: border-box;
```

Comment:
It is a good practice to add `box-sizing: border-box;` on all elements `* {...}`